ui/00_message/ui/system_log.toml

```diff
@@ -1461,58 +1461,88 @@ old = '''
 [[message]]
 key = 'syslog_action_evildra_01'
 old = 'マグマから炎の力を吸収しようとしている――'
+new = 'Trying to absorb the power of flames from magma'
```

Comment:
Please correct the punctuation of added messages in this file.

Reply:
I added a full stop to all of the added messages. But I also noticed there're some messages missing them as well. Should I add it to them too?

Reply:
If you'd like to do a pass of messages missing proper punctuation that would be fine as a new PR, otherwise I will probably do a pass over it myself. I won't catch every issue (and probably cause my own from times) so an extra eye will be helpful.